db/dbformat.h

```diff
@@ -31,6 +31,8 @@ static const size_t kL0_CompactionTrigger = 6;
 // Level-0 (any overlapped level) number of files where a grooming
 //     compaction could start
 static const size_t kL0_GroomingTrigger = 4;
+static const size_t kL0_GroomingTrigger10min = 2;
```

Comment:
It depends on the compiler and linker, but I have seen static variables that are declared and instantiated in a header file cause an instance of the variable to be statically instantiated in each source file that includes the header file. It's not a big deal since the variables are also const, but it does add extra space by duplicating the variables in multiple source files.

In this case, I am cool with not changing anything.